source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -179,6 +179,15 @@ primary shard to each database by means of a round-robin.
    restart the migration from the start. For more information, see
    :ref:`c2c-sharded-limitations`. 
 
+Embedded Config Server Shards
+'''''''''''''''''''''''''''''
+
+Starting in 8.0. MongoDB introduces support for embedded config server shards.
```

Comment:
[nit/typo] I believe the second period should be a comma. 
```suggestion
Starting in 8.0, MongoDB introduces support for embedded config server shards.
```

Reply:
Also it might be helpful to link users to the [config shard concept page](https://www.mongodb.com/docs/manual/core/config-shard/#config-shard), so they can learn more.